### PR TITLE
Added release flow for community products

### PR DESCRIPTION
**Pull request by Korulag**

### CODE REVIEW

File: alws/alembic/versions/ec505e94fd1a_added_community_release_flow.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Revision ID: ec505e94fd1a
+Revises: 26b384d3493f
+Create Date: 2022-08-18 07:38:00.361373
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from alws.constants import DEFAULT_PRODUCT
```
Comment (anfimovdm) [inline alws/alembic/versions/ec505e94fd1a_added_community_release_flow.py:11]: unused import

File: alws/alembic/versions/46393610f521_rbac_for_signing_and_releasing.py [hunk 2]
```diff
@@ -33,11 +33,13 @@ def upgrade():
         sa.PrimaryKeyConstraint('platform_id', 'role_id')
     )
     op.add_column('build_releases', sa.Column('team_id', sa.Integer(), nullable=True))
+    op.create_foreign_key('build_releases_team_id_fkey', 'build_releases', 'teams', ['team_id'], ['id'])
     # ### end Alembic commands ###
 
 
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('build_releases_team_id_fkey', 'build_releases', type_='foreignkey')
```
Comment (anfimovdm) [inline alws/alembic/versions/46393610f521_rbac_for_signing_and_releasing.py:42]: can we insert these changes in new migration instead?

File: alws/release_planner.py [hunk 3]
```diff
@@ -170,6 +233,393 @@ async def get_pulp_packages(
...
+                    models.User.roles).selectinload(
+                    models.UserRole.actions),
+                selectinload(models.Product.team).selectinload(
+                    models.Team.roles).selectinload(
+                    models.UserRole.actions),
+                selectinload(models.Product.roles).selectinload(
+                    models.UserRole.actions),
+                selectinload(models.Product.repositories)
+            )
+        )).scalars().first()
```
Comment (anfimovdm) [inline alws/release_planner.py:281]: i believe that we have query like this in `crud/products`

File: alws/release_planner.py [hunk 4]
```diff
@@ -170,6 +233,393 @@ async def get_pulp_packages(
...
+                models.Team.roles).selectinload(models.UserRole.actions),
+            selectinload(models.Release.platform).selectinload(
+                models.Platform.repos.and_(
+                    models.Repository.production.is_(True)
+                ),
+            ),
+            selectinload(models.Release.product).selectinload(
+                models.Product.repositories
+            ),
+        ).with_for_update()
```
Comment (anfimovdm) [inline alws/release_planner.py:423]: can we move this query in function and use it here and in `update_release`?

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/alembic/versions/ec505e94fd1a_added_community_release_flow.py (hunk 1 above)
Comment (Korulag) [reply alws/alembic/versions/ec505e94fd1a_added_community_release_flow.py]: Fixed

File: alws/alembic/versions/46393610f521_rbac_for_signing_and_releasing.py (hunk 2 above)
Comment (Korulag) [reply alws/alembic/versions/46393610f521_rbac_for_signing_and_releasing.py]: Moved

File: alws/release_planner.py (hunk 3 above)
Comment (Korulag) [reply alws/release_planner.py]: Thanks, re-used existing function

File: alws/release_planner.py (hunk 4 above)
Comment (Korulag) [reply alws/release_planner.py]: Moved

**Review by javihernandez — APPROVED** [review]

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**